components/permit-holders/modals/PreviousDoctorsInformationModal.tsx

```diff
@@ -92,7 +86,7 @@ export default function PreviousDoctorsInformationModal({
           </ModalHeader>
           <ModalBody paddingY="0px" paddingX="4px">
             <Box>
-              <Table columns={COLUMNS} data={DATA} />
+              <Table columns={COLUMNS} data={previousPhysicianData || []} />
```

Comment:
```suggestion
              <Table columns={COLUMNS} data={previousPhysicianData} />
```
Don't need `|| []` since the prop is mandatory. Empty array is truthy in JS